sunpy/timeseries/sources/noaa.py

```diff
@@ -202,7 +199,7 @@ def peek(self, **plot_args):
 
             import sunpy.timeseries
             import sunpy.data.sample
-            noaa = sunpy.timeseries.TimeSeries(sunpy.data.sample.NOAAPREDICT_TIMESERIES, source='NOAAPredictIndices')
+            noaa = sunpy.timeseries.TimeSeries('ftp://ftp.swpc.noaa.gov/pub/weekly/Predict.txt', source='NOAAPredictIndices')
```

Comment:
I think this needs a remote data flag?

Reply:
Looks like that might be causing the doc build failure.

Reply:
?
This is in a `plot` directive, not in a doctest.  There's no such thing as a remote-data flag.
The doc build is supposed to grab remote data.  Even if there were such a control flag, it'd be enabled for the doc build.

Reply:
The doc build passed on a re-run, which suggests that switching to online retrieval may have made our online builds even more prone to failure.